coderd/rbac/roles_test.go

```diff
@@ -888,6 +888,21 @@ func TestRolePermissions(t *testing.T) {
 				},
 			},
 		},
+		{
+			Name:     "AIBridgeInterceptions",
+			Actions:  []policy.Action{policy.ActionCreate, policy.ActionRead, policy.ActionUpdate},
+			Resource: rbac.ResourceAibridgeInterception,
+			AuthorizeMap: map[bool][]hasAuthSubjects{
+				true: {owner},
+				false: {
+					memberMe, orgMemberMe, otherOrgMember,
+					orgAdmin, otherOrgAdmin,
+					orgAuditor, otherOrgAuditor,
+					templateAdmin, orgTemplateAdmin, otherOrgTemplateAdmin,
+					userAdmin, orgUserAdmin, otherOrgUserAdmin,
+				},
+			},
+		},
```

Comment:
The actual resource uses `ownerID`. So this would match the production case. Is this permission assertion correct to how you want it?

```suggestion
		{
			Name:     "AIBridgeInterceptions",
			Actions:  []policy.Action{policy.ActionCreate, policy.ActionRead, policy.ActionUpdate},
			Resource: rbac.ResourceAibridgeInterception.WithOwner(currentUser.String()),
			AuthorizeMap: map[bool][]hasAuthSubjects{
				true: {owner, memberMe, orgMemberMe},
				false: {
					otherOrgMember,
					orgAdmin, otherOrgAdmin,
					orgAuditor, otherOrgAuditor,
					templateAdmin, orgTemplateAdmin, otherOrgTemplateAdmin,
					userAdmin, orgUserAdmin, otherOrgUserAdmin,
				},
			},
		},
```

Reply:
Good catch, thanks 👍